packages/jupyter-chat/src/model.ts

```diff
@@ -166,6 +172,26 @@ export interface IChatModel extends IDisposable {
    */
   focusInput(): void;
 
+  /**
+   * Add attachment to the next message to send.
+   */
+  addAttachment?(attachment: IAttachment): void;
+
+  /**
+   * Remove attachment to the next message to send.
+   */
+  removeAttachment?(attachment: IAttachment): void;
+
+  /**
+   * Update attachments.
+   */
+  updateAttachments?(attachments: IAttachment[]): void;
```

Comment:
This only gets used to clear the list of attachments, and there isn't a clear use-case for bulk updates. Given this, can you rename this method to `clearAttachments()` and update the implementation accordingly?

If adding/removing multiple files is required in the future, that capability should be provided by new `addAttachments()` and `removeAttachments()` methods which accept a list of attachments to add/remove.